salt/kube-apiserver/apiserver.jinja

```diff
@@ -34,6 +34,9 @@ KUBE_API_ARGS="--advertise-address={{ salt.caasp_net.get_primary_ip() }} \
                --cloud-config=/etc/kubernetes/openstack-config \
   {%- endif -%}
 {% endif %}
+{%- if pillar['feature_gates'] %}
+               --feature-gates={{ salt.caasp_pillar.get_feature_gates_flags() }} \
```

Comment:
I can do that, but this would require to:

  * Introduce a new `FEATURE_GATES` environment variable inside of `/etc/kubernetes/config`
  * Change all the kubernetes package to have all the systemd units  reference the`FEATURE_GATES` environment variable on their `ExecStart`. 

Is everybody fine with that?

Reply:
I think it would be better to create a `KUBE_COMMON_ARGS=` in /etc/kubernetes/config, and then update the services to use it - that way all common options can move there

Reply:
Good idea. I'll take care of that